getting-started.adoc

```diff
@@ -1,3 +1,251 @@
+
+IMPORTANT: It's *crucial* the information you enter here is correct. Your trading partner can only acknowledge your payment if it's from the account detailed here. Once you've created an account, you cannot edit its information (if you need to make a change, you'll have to delete and add the account again).
+
+Once you're done, hit the `Save new account` button and go back to the `Buy BTC` screen. Offers requiring the payment method you just configured should now be enabled:
+
+.After configuring a payment account: we picked Zelle, so Zelle offers are now enabled.
+image::after-adding-fiat-account.png[Configuring a national currency account]
+
+== Fund your Bisq wallet
+
+There's just one more thing to do before we can trade with Bisq: we need to send enough bitcoin to our Bisq wallet to cover a security deposit as well as trading and mining fees.
+
+To prevent fraud, Bisq requires both buyers and sellers to post a security deposit in bitcoin for every trade. On the buyer's side, this amount is usually {btc-deposit} BTC, plus a trading fee and mining fees for the Bitcoin transactions involved in each trade.
+
+As of this writing, that means you'll need {btc-deposit-plus-fees} BTC.
+
+If you don't already own this much bitcoin, you'll need to obtain it. How? You've got options. Friends and family are often the best option—simply buy a small amount of bitcoin directly and in person from someone you trust. Otherwise, you can try a https://coinatmradar.com/[local bitcoin ATM] (many don't require ID for small transactions), https://www.meetup.com/topics/bitcoin/[bitcoin meetups/events], or vouchers.
+
+[WARNING]
+.How about getting bitcoin through a centralized exchange like Coinbase?
+====
+Most centralized platforms and exchanges (like Coinbase, Binance, Kraken, etc) track your personal information, putting you at risk by tying your identity to the bitcoin you buy and sell there. And because Bitcoin transactions are public and easily traceable, this means that potentially _all_ of your future transactions involving those bitcoin can be traced back to you.
+
+Bisq is built from the ground up to avoid this privacy fiasco, so we strongly recommend avoiding centralized exchanges and to get your first bitcoin through one of the other channels mentioned above.
+====
+
+Once you've gotten enough bitcoin for your security deposit and fees, you can move it into Bisq's built-in wallet. This isn't strictly required, but it'll make trading more convenient since the security deposit & fees you need to accept an offer will be readily available when making or taking an offer.
+
+[IMPORTANT]
+====
+Make sure to password-protect your Bisq client and to write down the _seed words_ for your Bisq wallet. Seed words allow you to recover the funds in your wallet in case of disaster. You'll find them in the `Wallet seed` section of the `Account` screen. https://docs.bisq.network/secure-wallet.html[Learn more here].
```

Comment:
I think linking to a more substantive document is the way to go for this. Otherwise this document will get even longer than it already is...plus securing wallets is so crucial that it deserves to stand on its own as a page.

Perhaps we can find a way to make that page useful & general enough to serve as advice for new bitcoin users no matter what wallet they use (i.e., to optimize SEO and inbound traffic).